src/middleware/graphql/graphqlish.service.js

```diff
@@ -0,0 +1,161 @@
+const { parse } = require('graphql');
+const { search } = require('../../services/base/base.service');
+const { getCurrentDate, mapEOB } = require('./graphqlish.mappers');
+
+function filter(eobData, billablePeriod, allowedAmount, youPaid, insurancePaid, claimTypes) {
+  let filteredData = eobData.entry;
+  filteredData =
+    filteredData === null || filteredData === void 0
+      ? void 0
+      : filteredData.filter((bundleEntry) => {
+          var _a, _b;
```

Comment:
If it's not a requirement for a library, could we use a different variable name for this?